src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -80,165 +80,6 @@
                                            (rf/dispatch [:wallet/update-receiver-networks
                                                          chain-ids]))}]))}]))
 
-#_(defn- edit-amount
```

Comment:
We will never need this since send screen is going to be simplified

Reply:
Why can't we simply remove it then?